cpp/dolfinx/geometry/BoundingBoxTree.cpp

```diff
@@ -34,8 +34,8 @@ std::vector<std::int32_t> range(const mesh::Mesh& mesh, int tdim)
 }
 //-----------------------------------------------------------------------------
 // Compute bounding box of mesh entity
```

Comment:
The data layout in `std::array<std::array<double, 3>, 2>` is quite evident, but not in `std::array<double, 6>`.
I understand that it uses `XYZXYZ` but had to look at the function internals.